book.bib

```diff
@@ -423,12 +423,90 @@ @software{irvine2022NewGraphEnvironmentfpr
+  doi = {10.1111/j.1365-2400.2010.00751.x},
+  url = {http://doi.wiley.com/10.1111/j.1365-2400.2010.00751.x},
+  urldate = {2020-12-30},
+  file = {C\:\\Users\\allan\\Zotero\\storage\\DSSHZCH3\\Kemp and O'Hanley - 2010 - Procedures for evaluating and prioritising the rem.pdf},
+  langid = {english},
+}
+
+@Article{shaw_etal2016Importancepartial,
+  title = {Importance of Partial Barriers and Temporal Variation in Flow When Modelling Connectivity in Fragmented River Systems},
+  author = {Edward A. Shaw and Eckart Lange and James D. Shucksmith and David N. Lerner},
+  year = {2016},
+  journaltitle = {Ecological Engineering},
+  shortjournal = {Ecological Engineering},
+  volume = {91},
+  pages = {515--528},
+  issn = {09258574},
+  doi = {10.1016/j.ecoleng.2016.01.030},
+  url = {https://linkinghub.elsevier.com/retrieve/pii/S0925857416300301},
+  urldate = {2020-12-23},
+  file = {C\:\\Users\\allan\\Zotero\\storage\\FPK43M3G\\Shaw et al. - 2016 - Importance of partial barriers and temporal variat.pdf},
+  langid = {english},
+}
+
+@Article{mahlum_etal2014EvaluatingBarrier,
+  title = {Evaluating the {{Barrier Assessment Technique Derived}} from {{FishXing Software}} and the {{Upstream Movement}} of {{Brook Trout}} through {{Road Culverts}}},
+  author = {Shad Mahlum and David Cote and Yolanda Wiersma and Dan Kehler and K. Clarke},
+  year = {2014},
+  journaltitle = {Transactions of the American Fisheries Society},
+  shortjournal = {Transactions of the American Fisheries Society},
+  volume = {143},
+  doi = {10.1080/00028487.2013.825641},
+  file = {C\:\\Users\\allan\\Zotero\\storage\\A8XXNI65\\Mahlum et al. - 2014 - Evaluating the Barrier Assessment Technique Derive.pdf},
+}
+
+@Article{kemp_ohanley2010Proceduresevaluating,
+  title = {Procedures for Evaluating and Prioritising the Removal of Fish Passage Barriers: A Synthesis: {{EVALUATION OF FISH PASSAGE BARRIERS}}},
+  shorttitle = {Procedures for Evaluating and Prioritising the Removal of Fish Passage Barriers},
+  author = {P. S. Kemp and J. R. O'Hanley},
+  year = {2010},
+  journaltitle = {Fisheries Management and Ecology},
+  pages = {no-no},
+  issn = {0969997X, 13652400},
+  doi = {10.1111/j.1365-2400.2010.00751.x},
+  url = {http://doi.wiley.com/10.1111/j.1365-2400.2010.00751.x},
+  urldate = {2020-12-30},
+  file = {C\:\\Users\\allan\\Zotero\\storage\\DSSHZCH3\\Kemp and O'Hanley - 2010 - Procedures for evaluating and prioritising the rem.pdf},
+  langid = {english},
+}
+
+@misc{washingtondepartmentoffishwildlife2009FishPassage,
+  title = {Fish {{Passage Barrier}} and {{Surface Water Diversion Screening Assessment}} and {{Prioritization Manual}}},
+  author = {{Washington Department of Fish \& Wildlife}},
+  date = {2009},
+  publisher = {{Washington Department of Fish and Wildlife. Olympia, Washington.}},
+  url = {https://wdfw.wa.gov/sites/default/files/publications/00061/wdfw00061.pdf},
+  urldate = {2021-02-13}
+}
+
+@Article{mazany-wright_etal2021BulkleyRiver,
```

Comment:
Is this referenced somewhere in this document?  I don't think it should be since this is from the Bulkley.  I guess I might have put it in there..... seems odd. Please file an issue if it is there somewhere unless it is really helpful for the reader.

Reply:
Ya it's in the methods here https://github.com/NewGraphEnvironment/fish_passage_peace_2022_reporting/blob/d0545061987c8ab48dd13cb5fb7d99f56d4fd237/0300-methods.Rmd#L29. I did take this reference out at first but then put it back in because I thought it might still be helpful. As it describes the big picture of what we're building.

Reply:
To be fair - left text, added Pacific Salmon Foundation and pulled reference. Pushed to repo just now